WooCommerce/UITestsFoundation/Screens/Payments/PaymentsScreen.swift

```diff
@@ -41,6 +61,33 @@ public final class PaymentsScreen: ScreenObject {
         return self
     }
 
+    public func tapCollectPayment() throws -> Self {
+        collectPaymentButton.tap()
+        return self
+    }
+
+    public func enterPaymentAmount(_ amount: String) throws -> Self {
+        app.enterText(text: amount)
+        nextButton.tap()
+        return self
+    }
+
+    public func takeCashPayment() throws -> Self {
+        takePaymentButton.waitAndTap()
+        cashPaymentButton.waitAndTap()
+
+        markAsPaidButton.tap()
+        return self
+    }
+
+    @discardableResult
+    public func verifyOrderCompletedToastDisplayed() throws -> Self {
+        let orderCompletedToast = app.staticTexts.matching(NSPredicate(format: "label CONTAINS 'Reader'")).firstMatch
```

Comment:
😅 

```suggestion
        let orderCompletedToast = app.staticTexts.matching(NSPredicate(format: "label CONTAINS 'Order completed'")).firstMatch
```

Reply:
😳 whoops! don't know how i missed this! will update!